src/reducers/MapReducer.js

```diff
@@ -56,7 +56,7 @@ const INITIAL_STATE = {
     viewStyle: map.SearchOptions.MAP,
     sites: [],
     displaySites: [],
-    filterCriteriaKeys: {accessibility: [], facilities: [], price: []}
+    filterCriteriaKeys: {accessibility: [], facilities: [], price: [], features: []}
```

Comment:
I still have to fix the filter logic, but adding `features` is in preparation for that.